### PR TITLE
New management command

### PR DESCRIPTION
**Pull request by reginafcompton**
This PR adds a simple management command for denying applicants for reasons not specified in the standard admin UI.

### CODE REVIEW

File: lots_admin/management/commands/denial_special.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import json
+import logging
+
+from django.core.management.base import BaseCommand, CommandError
+
+from lots_admin.models import ApplicationStatus, DenialReason, Review, User
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument('--pins_with_denials',
```
Comment (hancush) [inline lots_admin/management/commands/denial_special.py]: just want to confirm: this management command is to deny all applications for a given lot?

Comment (reginafcompton) [reply lots_admin/management/commands/denial_special.py]: yep. the lots are not available for sale for various reasons (e.g., being held for a Redline Extension), but were listed on the site.

**Comment by reginafcompton** [comment]
haha! Sure. Thought I thought "denial special" sounded like something tasty you could get in a dimly-lit diner off the interstate.

**Comment by hancush** [comment]
sounds... yummy.......

**Comment by reginafcompton** [comment]
all right - new communicative, albeit lengthy command name

**Pull request merged**